paf-mvp-core-js/tests/test-cases/node.test.ts

```diff
@@ -0,0 +1,196 @@
+import { IJsonValidator, JsonSchemaType, JsonValidation } from '@core/validation/json-validator';
+import { IdentityConfig, Node } from '@core/express';
+import { createRequest, createResponse, MockResponse } from 'node-mocks-http';
+import { NextFunction, Response } from 'express';
+import { NodeError, NodeErrorType } from '@core/errors';
+import { getTimeStampInSec } from '@core/timestamp';
+import { encodeBase64, QSParam } from '@core/query-string';
+
+const buildStaticJsonValidator = (validationResult: boolean): IJsonValidator => {
+  const jsonValidation: JsonValidation = {
+    isValid: validationResult,
+    value: {},
+    errors: validationResult ? [] : [new Error('error message from validator')],
+  };
+  return {
+    start: jest.fn(),
+    validate: jest.fn((schema: JsonSchemaType, jsonStr: string) => jsonValidation),
```

Comment:
Can you remove the warning, please? I am not sure if it is possible without disabling the linter on this specific line.

Reply:
removed !